packages/react-instantsearch-core/src/connectors/connectDynamicWidgets.ts

```diff
@@ -13,6 +13,11 @@ export default createConnector({
     transformItems: PropTypes.func,
   },
 
+  // count as a widget for server-side rendering
```

Comment:
Here's a suggestion, if I understood correctly:

```suggestion
  // This is required to flag as a widget for SSR to pick up its search parameters.
```